
# Exercício 1
## Parte 1
Crie 3 variáveis:

nomePrimeiro, nomeMeio, nomeUltimo

Atribua um valor para cada uma delas e imprima a seguinte frase para o usuário:

"Gabriel Gonçalves de Briro, seja muito bem vindo a DNC"

Não precisa usar input( )

In [28]:
nomePrimeiro= 'Gabriel '
nomeMeio= 'Gonçalves de '
nomeUltimo= 'Brito'

print(nomePrimeiro + nomeMeio + nomeUltimo + ', seja bem vindo a DNC')

Gabriel Gonçalves de Brito, seja bem vindo a DNC


## Parte 2
Agora peça para o usuário digitar as 3 variáveis antes de mostrar a mesma frase da parte 1.

Resultado esperado

In [29]:
nomePrimeiro=input('Qual o seu primeiro nome?\n')
nomeMeio=input('Qual o seu nome do meio?\n')
nomeUltimo=input('Qual o seu ultimo nome?\n')

print(nomePrimeiro, nomeMeio, nomeUltimo + ', seja muito bem vindoa DNC')

Gabriel Gonçalves de Brito, seja muito bem vindoa DNC


# Exercício 2
#### 1. Vamos criar nosso primeiro script 💪, que:

Armazene em uma variavel o nome do cliente, a variável precisa se chamar cliente_nome e o nome do cliente você pode usar sua criatividade.

Crie uma variável que armazene a idade do cliente, a variável precisa se chamar cliente_idade.

Exiba na tela as 2 variáveis e apenas uma variável por linha. Ex: "O nome do Cliente é: Nathan Pereira".

In [24]:
cliente_nome='Gabriel Gonçalves de brito'
cliente_idade= 25

print('Nome do cliente:', cliente_nome)
print('A idade do cliente:', cliente_idade)

Nome do cliente: Gabriel Gonçalves de brito
A idade do cliente: 25


#### 2. Agora que você já criou seu primeiro script vamos aumentar a dificuldade!
Agora é o cliente que vai colocar as informações! Crie uma variável que receba como input o nome e a idade do cliente!
Para ajudar o usuário exiba, essas variáveis na tela.

In [3]:
cliente= input('Qual seu nome?\n')
idade= input('Qual a sua idade?\n')

print('O nome do clinte é:', cliente)
print('A idade do cliente é:', idade)

O nome do clinte é: Gabriel Gonçalves de Brito
A idade do cliente é: 25


#### 3. Vamos brincar com mais variáveis agora?!
Crie uma variável do tipo FLOAT que vai armazenar o valor do produto, que receberá o valor de 25.30!
Crie uma variável do tipo BOOL que vai armazenar se ele é ou não um cliente premium, recebendo o valor de True.
Agora exiba essas variáveis na tela.
Exiba também o tipo dessas variáveis.

In [20]:
produto_valor=25.30
cliente_premium=True

print('O valor do produto é:', produto_valor)
print ('O tipo da variável produto-valor é:', type( produto_valor))
print('O cliente é premium?', cliente_premium)
print('O tipo da variável cliemte_premium é:', type(cliente_premium))
print()

print('o valor do produto é: ,produto_valor', produto_valor, type (produto_valor))
print('o cliente é premium?', cliente_premium, type(cliente_premium))

O valor do produto é: 25.3
O tipo da variável produto-valor é: <class 'float'>
O cliente é premium? True
O tipo da variável cliemte_premium é: <class 'bool'>

o valor do produto é: ,produto_valor 25.3 <class 'float'>
o cliente é premium? True <class 'bool'>


### tipos de dados

*str:* cadeia de caracteres= ']~´[@', '7', 'Gabriel'

*int:* inteiro= 5, 10, 25

*float:* real= 5.3, 25.3

*bool:* booleanos= true, false


### funções

**print()=** imprimr uma frase

**type()=** verifica o tipo de dado de uma variável

**input()=** pede pro usuário fornecer um valor (str)